fhirclient/models/fhirsearch.py

```diff
@@ -110,59 +109,65 @@ def handle(self):
         """
         handler = FHIRSearchParamHandler.handler_for(self.name)(None, self.value)
         return handler.handle(self.copy())
-    
+
     def as_parameter(self):
         """ Return a string that represents the reciever as "key=value".
         """
+        if isinstance(self.value, list):
+            return_str = ""
+            for val in self.value:
+                if return_str:
+                    return_str += '&'
+                return_str += '{}={}'.format(self.name, quote_plus(val, safe=',<=>'))
+            return return_str
```

Comment:
I like it!  A one-liner version to do the same thing, you could replace 117 - 121 with:
`return "&".join(['{}={}'.format(self.name, quote_plus(val, safe=',<=>')) for val in self.value]`

Reply:
made it even better